test/Sentry.Tests/Internals/SentryStackTraceFactoryTests.cs

```diff
@@ -191,7 +191,7 @@ public void FileNameShouldBeRelative()
         var path = Path.Combine("Internals", "SentryStackTraceFactoryTests.cs");
         Assert.Equal(path, frame.FileName);
 
-        var fullPath = GetThisFilePath();
+        var fullPath = "test/Sentry.Tests/Internals/SentryStackTraceFactoryTests.cs";
```

Comment:
@vaind could this have any side effects? I think this test was more about the `frame.FileName` property above (making sure this was a combination of Internals + filename) but I'm not sure why that's important or what the intent was here.